homeassistant/components/notion/__init__.py

```diff
@@ -322,6 +324,14 @@ def listener(self) -> Listener:
         """Return the listener related to this entity."""
         return self.coordinator.data.listeners[self._listener_id]
 
+    @callback
+    def _async_get_bridge(self, bridge_id: int) -> Bridge | None:
+        """Get a bridge by ID (if it exists)."""
+        if (bridge := self.coordinator.data.bridges.get(bridge_id)) is None:
+            LOGGER.info("Entity references a non-existent bridge ID: %s", bridge_id)
```

Comment:
Shouldn't we log this with Warning? And should we mention more about the entity? Can people do something about this?

Reply:
A warning doesn't make sense here, since the user can't do anything about it—it's a bug inside Notion's API. To that end, I don't know that mentioning more about the entity gives us anything material to work with.

Reply:
Is it right to log on info then?

Like I see Martin comment regularly that we shouldn't use log.info, hence I'm a bit hesitant about this one. Wdyt?

Reply:
Hmm, not sure. It's still useful to us, if not the user. Would DEBUG be better? I'm open!

Reply:
If it's only useful for developers, we should log it at debug level. :+1:

Reply:
And otherwise?

Reply:
It depends what the message is. Errors are logged at error level, warnings at warning level and so on.